app/forms/candidate_interface/other_qualification_details_form.rb

```diff
@@ -179,5 +179,11 @@ def sanitize_grade_where_required
         self.grade = grade.delete(' ').upcase if grade
       end
     end
+
+    def should_validate_grade?
+      (qualification_type != OtherQualificationTypeForm::NON_UK_TYPE &&
+          qualification_type != OtherQualificationTypeForm::OTHER_TYPE) ||
+        other_uk_qualification_type == 'BTEC'
```

Comment:
could wrap this in a `btec?` method, and us it in the view template below as well

Reply:
Really neat idea, cheers